.gitignore

```diff
@@ -16,3 +16,4 @@ bundles
 dist
 **/*.egg-info
 .vscode
+.venv
```

Comment:
I always push back gently about additions like this to gitignore files.

If you commonly create ".venv" in projects where you work, it's better if you use your own global gitignore file, because it'll allow git to ignore ".venv" in any project on your system, without modifying the project gitignore file.

Here's a quick read that shows how to set it up: https://sebastiandedeyne.com/setting-up-a-global-gitignore-file/

Reply:
Oooh - didn't know about that. Thanks.